### DISCUSSION TITLE
Development Environment

### DISCUSSION THREAD

**Discussion by hubing53** [AvaloniaUI/Avalonia#9041] [Q&A]
Can you help me? How to develop the app with Visual studio for mac 2022 Bacause of the JetBrain Rider can't on Mac that is Vitual System。 when i open the .axaml file not previrew the axaml View
![1664351265321_216654ae](https://user-images.githubusercontent.com/112156001/192720168-d830b0ef-7a86-4726-95da-4ce1bdd604ee.png)

**Comment by rabbitism** [comment]
Previewer works via VS extension. Is it allowed to install that extension in VS for mac?

**Reply by hubing53** [reply]
visual studio for  Window11 and visual studio for mac is different project

**Comment by kekekeks** [comment]
I don't think we support running Avalonia in macOS virtual machines without hw graphics acceleration. Note that the next version will require Metal support.
